sw/device/silicon_creator/rom_ext/sival/BUILD

```diff
@@ -46,7 +46,7 @@ opentitan_binary(
         "//sw/device/lib/crt",
         "//sw/device/silicon_creator/lib:manifest_def",
         "//sw/device/silicon_creator/rom_ext",
-        "//sw/device/silicon_creator/rom_ext/keys/fake",
+        "//sw/device/silicon_creator/rom_ext/sival/keys",
```

Comment:
Don't we still want the fake keys available here?

Reply:
We want the sival and prodc ROM_EXTs to be able to run application binaries signed with their respective keys.  The fake keys are for the "default" ROM_EXT and allow test programs (such as the rom_ext/e2e tests) to run.

A "fake signed sival rom_ext" is signed with the fake ROM verification keys, thus allowing it to boot in the FPGA, but that ROM_EXT should be able to then boot a sival-signed application binary.